Tests/BetterCodableTests/DefaultNilTests.swift

```diff
@@ -0,0 +1,63 @@
+import XCTest
+import BetterCodable
+
+class DefaultNilTests: XCTestCase {
+    /// This test demonstrates the problem that `@DefaultNil` solves. When decoding
+    /// optional types, it often the case that we end up with an error instead of
+    /// defaulting back to `nil`.
+    func testDecodingBadUrlAsOptionalWithoutDefaultNil() {
+        struct Fixture: Codable {
+            var url: URL?
+        }
+        let jsonData = #"{"url":""}"#.data(using: .utf8)!
```

Comment:
Let's use something besides an empty string that demonstrates a malformed URL resulting in `nil.`